components/GroupCard.tsx

```diff
@@ -0,0 +1,42 @@
+import Tag from '../components/Tag';
+import Link from 'next/link';
+
+type Props = {
+  title: string;
+  subtitle?: string;
```

Comment:
Same there, maybe use `nbMembers` or something like this to be more explicit. Why is it nullable ? Btw, by convention, Nullable props are given at the end of the object